deploy/src/main/config/hono-adapter-amqp-vertx-config.yml

```diff
@@ -43,15 +43,16 @@ hono:
     keyPath: ${secret.path}/amqp-adapter-key.pem
     certPath: ${secret.path}/amqp-adapter-cert.pem
     trustStorePath: ${secret.path}/trusted-certs.pem
-  metric:
-    reporter:
-      graphite:
-        active: true
-        host: influxdb.hono
-        port: 2003
-# disable spring boot actuator endpoints, enable metrics
+  metrics:
+    legacy: true
+
+# disable spring boot actuator endpoints
 endpoints:
   enabled: false
-  metrics:
-    enabled: true
 
+management:
```

Comment:
it looks  like configuration of the reporter has changed when compared to https://www.eclipse.org/hono/admin-guide/monitoring-tracing-config/ 

IMHO you should update the documentation accordingly (and add a corresponding note to the release notes as well)